racket/src/cs/thread.sls

```diff
@@ -52,4 +80,8 @@
 
   (set-scheduler-lock-callbacks! (lambda () (1/make-semaphore 1))
                                  1/semaphore-wait
-                                 1/semaphore-post))
+                                 1/semaphore-post)
+
+  (set-future-callbacks! 1/future? 1/current-future
+  			 future-block current-future-prompt))
+			 
```

Comment:
Please fix ending (to end in a newline).

Reply:
I'm not sure I understand this.  You would like the last empty line to be removed?

Reply:
Yes, remove the whitespace at the end, especially because it isn't followed by a newline. (By old Unix rules, a text file isn't properly formed unless the last character is a newline. That rule is often ignored now, but it's still a good guideline.)